From: Modestas Vainius <modax@debian.org>
Subject: set sane timestamps when generating header symlinks for non-existing headers
Forwarded: no
Origin: vendor
Last-Update: 2010-12-05

This is particularly important for QtCore/QtConfig symlink because typically
qconfig.h does not exist when syncqt is run by ./configure. So instead of
assigning a modification timestamp equal to the epoch (00:00 January 1, 1970
GMT.) for the symlink, just use current time().

---
 bin/syncqt |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/bin/syncqt
+++ b/bin/syncqt
@@ -848,7 +848,7 @@ foreach my $lib (@modules_to_sync) {
                             print "SYMBOL: $_\n";
                         }
                     } else {
-                        my $ts = (stat($iheader))[9];
+                        my $ts = (-e $iheader) ? (stat($iheader))[9] : time;
                         #find out all the places it goes..
                         my @headers;
                         if ($public_header) {
